Product tour (#3065)
ezsystems/developer-documentation: docs/administration/back_office/customize_integrated_help.md

- by modifying a link to user documentation
- by subscribing to the `ibexa_integrated_help.menu_configure.help_menu` event

## Disable integrated help features

> After you have installed the integrated help package, you can disable the entire feature or specific features on the system level.

Reviewer: ```suggestion
After you have installed the integrated help package, you can disable the entire package or specific features on the system level.
```
Alternatively, use "feature" for the overall set pprovided by the [ackage, and functionalities for the patrs. This would require updating line 18, 22, 31 and other occurrences by replacing "features" with "functionalities":

```suggestion
After you have installed the integrated help package, you can disable the entire feature or specific functionalities on the system level.
```

Author: I went with feature/functionalities - not a fan of "package", because sometimes a feature has more than 1 package (Composer package)

### Disable all features

> To disable both the Help center and the Product tour globally, for example, to run UI tests in a `dev` [environment](environments.md), in `config/packages` create the `ibexa_integrated_help.yaml` file with the following configuration:

Reviewer: ```suggestion
To disable both the Help center and the Product tour globally, for example, to run UI tests in a `dev` [environment](environments.md), in `config/packages`, create the `ibexa_integrated_help.yaml` file with the following configuration:
```

``` yaml
ibexa_integrated_help:
    enabled: false
```

### Disable features independently

> To disable only the Help center or only the Product tour features, use the dedicated feature flags as in the example below:

Reviewer: ```suggestion
To disable only the Help center or only the Product tour functionalities, use the dedicated flags as in the example below:
```
See comment for line 20
